starters/features/leaflet-map/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "description": "Use Leaflet Maps in your Qwik app",
+  "__qwik__": {
+    "displayName": "Integration: Leaflet Maps",
+    "priority": -10,
+    "viteConfig": {},
+    "docs": [
+      "https://leafletjs.com/reference.html",
+      "https://leafletjs.com/examples/quick-start/",
+      "https://www.sitepoint.com/leaflet-create-map-beginner-guide/"
+    ]
```

Comment:
Can you add "nextSteps" section please? 
you can take inspiration from the other starter. Thanks.

Reply:
Perfect @gioboa ! 

It is the first time that I contribute to something as big as a framework and there are details that escape me. I'm sure that in the following contributions I won't bother so much. Thanks for all the suggestions for improvement :)